client/vertex-client.ts

```diff
@@ -38,7 +38,7 @@ interface CtorArgs {
   token: OAuth2Token;
 }
 
-const TokenExpiryBufferMs = 60000;
+const TenMinsInMs = 600_000;
 const SecToMs = 1000;
```

Comment:
Increase this from one to ten minutes since Badri mentioned he got a 401 when uploading files one by one.